src/components/home/model-selector.tsx

```
@@ -0,0 +1,73 @@
+import React from "react";
+import { Input } from "@/components/ui/input";
+import { ChevronDown } from "lucide-react";
+import { providerData } from "./constants";
+import { ISettings } from "@/types/game";
+import { useGame } from "@/contexts/game-context";
+
+interface ModelSelectorProps {}
+
+export function ModelSelector({}: ModelSelectorProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)